CONTRIBUTING.md

```diff
@@ -1,174 +1,315 @@
 # Contributing to Apify Documentation
 
-## Architecture
+## Before you start
 
-Apify documentation consists of the following top-level areas:
+1. Review this guide completely
+2. Setup you development environment
+3. Familiarize yourself with our style guide
```

Comment:
I'd namely mention **writing** style guide. "Style guide" may  as well refer to a coding style guide e.g. [Airbnb's JavaScript style guide ](https://github.com/airbnb/javascript)

Reply:
Isn't it obvious from the context? We are after all within documentation repository, and are reading contributing guidelines for said documentation?
I guess we can err on the safety side and call it `Documentation style guide?`

Reply:
I'd love that. I remember when I read it for the first time it made me think about what coding style guide I'll need to adopt to contribute to this docs. I remember slight confusion because I couldn't wrap my head around what coding I'll even need to do to contribute to the documentation. I just came to edit Markdown. 

I admit it might be my PTSD from the learning experience with other codebases, and I'm leaving it up to your consideration. It's just my $0.02 :)

Reply:
I don't see any harm in clarifying it :)